pkg/kubectl/cmd/drain_test.go

```diff
@@ -32,12 +32,13 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
+	"k8s.io/api/core/v1"
```

Comment:
after having a discussion in slack #sig-api-machinery, I think so far this is good, and interal type will only be covered in server side in the future, thanks you, this lgtm